packages/core/src/features/cohort/data/cohort_builder.json

```diff
@@ -92,11 +92,11 @@
     "treatment": {
       "label": "Treatment",
       "facets": [
-        "cases.diagnoses.best_overall_response",
         "cases.diagnoses.treatments.therapeutic_agents",
         "cases.diagnoses.treatments.treatment_intent_type",
         "cases.diagnoses.treatments.treatment_outcome",
-        "cases.diagnoses.treatments.treatment_type"
+        "cases.diagnoses.treatments.treatment_type",
+        "cases.diagnoses.best_overall_response"
```

Comment:
The resulting array is the same. I am not sure how this is working.

Reply:
It's a bit weird. Basically all the cards in this CB category are `cases.diagnoses.treatments.X` filters except `cases.diagnoses.best_overall_response` (note the different level of nesting). There's a BE issue where different ordering of these filters for aggregations could result in different numbers. So far, we found that the numbers returned are correct when the most nested fields are processed first. Ping me if you want more info. @paribartandhakal